EnglishFilter/sections/antiadblock.txt

```diff
@@ -414,6 +414,8 @@ clx.icu#@##ad-banner
 clx.icu#@#.ad-banner
 ! https://github.com/AdguardTeam/AdguardFilters/issues/36152
 @@||ijobanana.xyz^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/36371
+americanlookout.com###adtoniq-msgr-bar
```

Comment:
Please move to Annoyances/antiadblock.

Also instead of a new rule, you can add the domain to the f*ckadblock list rules.

```
americanlookout.com#@#.pub_300x250
americanlookout.com#@#.pub_300x250m
americanlookout.com#@#.pub_728x90
americanlookout.com#@#.text-ad
americanlookout.com#@#.text-ad-links
americanlookout.com#@#.text-ads
americanlookout.com#@#.textAd
americanlookout.com#@#.text_ad
americanlookout.com#@#.text_ads
@@||amazonaws.com/ads^$domain=americanlookout.com
```

Last rule not in f*ckadblock section, but in normal Annoyances/adblock.

Reply:
it is not critical. I see no tracking request.